lib/live/form.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative 'view'
+require 'trenni/builder'
+require 'json'
+
+module Live 
+    #Represents a Form within the view layer
+    #Differs from Live::View because the server sends both the 
+    #details and a serialized form from the view layer.
+    class Form < Live::View 
+        
+        #returns [String] javascript code to execute when the form is submitted.
+        def handleForm(details=false)
+            if details 
+                return "live.handleForm(this.id, event, #{details})"
```

Comment:
Did you introduce new method to live-js?

I believe we can just use `live.forward(this.id, event, new FormData(this))`.

Reply:
I implemented that with the user passing a details in mind, originally it was just going to be `live.handleForm(this.id, event)` but I thought maybe passing `details` had some use?

Reply:
I also added two new functions, one to prevent the form from submitting and refreshing the page, and another for serialization. [Here](https://github.com/socketry/live-js/pull/2/commits/5587cd980ba3c6984373bc1ee5efb4b717e6da94)